content/en/docs/languages/python/getting-started.md

```diff
@@ -21,7 +21,8 @@ Ensure that you have the following installed locally:
 ## Example Application
 
 The following example uses a basic [Flask](https://flask.palletsprojects.com/)
-application. If you are not using Flask, that's OK — you can use OpenTelemetry
+application. If you have not used Flask before or prefer a different framework,
+you can use OpenTelemetry
 Python with other web frameworks as well, such as Django and FastAPI. For a
 complete list of libraries for supported frameworks, see the
```

Comment:
What is suggested here seems fine, but it would be nice if we could have some consistency across languages. I proposed something in https://github.com/open-telemetry/opentelemetry.io/pull/3877/files, which adapted here, might read like:

```suggestion
application. You can use another web framework, such as Django and FastAPI.
For a complete list of libraries and supported frameworks, see the
```

WDYT @svrnm et al.?

Reply:
Works for me, also let's turn this in a short code soon

Reply:
@astanley-work please take a look, click on "commit suggestion" to directly accept @chalin's suggested changes